viv-wallet-app/src/services/login.ts

```diff
@@ -23,7 +23,7 @@ export class LoginService {
     private authorizations: Authorizations;
 
     constructor() {
-        const token = getCurrentToken();
+        const token = process.env.VUE_APP_DEV_JWT || getCurrentToken();
```

Comment:
how does it work... do I still need to login?
is it going to try to login?
maybe we need to document it somewhere